packages/plugin-flex/package.json

```diff
@@ -33,6 +33,7 @@
     "prebuild": "npm run clean",
     "build": "cd ../.. && tsc -p \"packages/plugin-flex/tsconfig.json\"",
     "clean": "rm -rf dist",
+    "link": "twilio plugins:link .",
```

Comment:
did you purposely add this for local dev?

Reply:
Ya